.github/workflows/build-docker.yaml

```diff
@@ -1,30 +0,0 @@
-name: build-docker
-
-on: 
-  workflow_run:
```

Comment:
This turned out to not do what I thought. This causes build-docker to run on the *master* branch whenever the pyright-ratchet job completes successful on *any* branch. That's not what we want.